src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -488,7 +552,7 @@ protected void processWorkUnitTemplate(
         try {
             // Merge data-model with template
             Template template = cfg.getTemplate(workUnit.getTemplateName());
-            File outputPath = new File(getDestinationDir() + "/" + workUnit.getTargetFileName());
+            File outputPath = new File(getDestinationDir() + File.separator + workUnit.getTargetFileName());
```

Comment:
Why do not use here just `new File(getDestinationDir(), workUnit.getTargetFileName())`, which use the default file separator?